common/persistence/visibility/store/elasticsearch/visibility_store.go

```diff
@@ -730,8 +735,11 @@ func (s *visibilityStore) convertQuery(
 	return namespaceFilterQuery, fieldSorts, nil
 }
 
-func (s *visibilityStore) setDefaultFieldSort(fieldSorts []*elastic.FieldSort) []elastic.Sorter {
+func (s *visibilityStore) setDefaultFieldSort(fieldSorts []*elastic.FieldSort, isScan bool) []elastic.Sorter {
```

Comment:
Since you're making changes to this function, this function should be called `getFieldSorter`.

Reply:
Yeah, I was wondering if it got re-purposed at some point. will do